templates/vue/src/components/TapestryApp.vue

```diff
@@ -50,7 +51,12 @@ export default {
   },
   mounted() {
     this.$root.$on("open-node", id => {
-      this.openNode(id)
+      let urlNodeId = id
+      const node = this.getNode(id)
+      if (node.presentationStyle === "units") {
+        urlNodeId = node.childOrdering[0]
+      }
+      this.openNode(urlNodeId)
```

Comment:
I think it would be more clear if you figure out and pass the correct node id of the actual page that you want to open when you emit the event rather than having to modify this file with this logic. I don't know why this component should get concerned with this logic.